src/authz-module/data/api.ts

```diff
@@ -8,12 +8,35 @@ export interface GetTeamMembersResponse {
   totalCount: number;
 }
 
+export interface PutTeamMembersResponse {
+  completed: { user: string; status: string }[];
+  errors: { user: string; error: string }[];
+}
+
+export interface AddTeamMembersRequest {
+  users: string[];
+  role: string;
+  scope: string;
+}
+
 // TODO: replece api path once is created
 export const getTeamMembers = async (object: string): Promise<TeamMember[]> => {
   const { data } = await getAuthenticatedHttpClient().get(getApiUrl(`/api/authz/v1/roles/users?scope=${object}`));
   return camelCaseObject(data.results);
 };
 
+export const addTeamMembers = async (
+  data: AddTeamMembersRequest,
+): Promise<PutTeamMembersResponse> => {
+  const res = await getAuthenticatedHttpClient().put(getApiUrl('/api/authz/v1/roles/users'), data);
+  return camelCaseObject(res.data);
+};
+
+export const getTeamRoles = async (libraryId: string): Promise<TeamRole[]> => {
```

Comment:
[enhancement] not blocking for the current iteration but this endpoint can be used for other scopes within the AuthZ module, a more generic parameter is better.